.editorconfig

```diff
@@ -0,0 +1,10 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+insert_final_newline = true
+
+[*.{json,yml}]
+indent_size = 2
```

Comment:
http://google.github.io/styleguide/javascriptguide.xml
从这里来看，大部分缩进是 2 个空格。

同一语句要折行时可以有多种选择：
var x = a ? b : c;  // All on one line if it will fit.

// Indentation +4 is OK.
var y = a ?
    longButSimpleOperandB : longButSimpleOperandC;

// Indenting to the line position of the first operand is also OK.
var z = a ?
            moreComplicatedB :
            moreComplicatedC;